cloudbuild.yaml

```diff
@@ -0,0 +1,23 @@
+steps:
+
+- name: 'docker/compose'
+  args: ['-f', './docker/openmrs-compose.yaml', '-f', './docker/sink-compose.yml',  'up', '-d']
+
+- name: 'adoptopenjdk/maven-openjdk8'
+  id: Test_Batch
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - 'apt-get update && apt-get install -y jq; cd bunsen; mvn --no-transfer-progress clean install; cd ../pipelines; mvn --no-transfer-progress clean install; cd ../; export ROOT_PATH=`pwd`; ./e2e-tests/batch-mode-tests.sh;'
```

Comment:
As we discussed, it would be nice to create two docker images for the pipelines and query lib. It does not need to be done in this PR but if you agree with the approach maybe add a TODO here or create an issue.

Reply:
The `e2e-tests` Dockerfile has all the dependencies needed to run both the query and pipeline tests